### ISSUE TITLE
Docstrings for twisted.trial.TestCase fail* methods were inconsistently capitalized

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2955]
|<img alt="amacleod's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| amacleod reported|
|-|-|
|Trac ID|trac#2955|
|Type|enhancement|
|Created|2007-12-14 23:19:30Z|

Some started with a lowercase letter, some with uppercase, and not all followed the pattern of "Fail if"/"Fail unless".

Also, there seemed to be some small factual errors in the docstrings.

Attachments:

* [trialdocstrings.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/1c3/1c3e8b8366570311e17244639fe5c699e1c65e66/ce10643c8f4cba293bf7d87b880a66d8405ec9c4.patch) (8381 bytes) - added by amacleod on 2007-12-14 23:20:23Z - patch for docstrings in twisted/trial/unittest.py

<details><summary>Searchable metadata</summary>

```
trac-id__2955 2955
type__enhancement enhancement
reporter__amacleod amacleod
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__trial trial
keywords__ 
time__1197674370000000 1197674370000000
changetime__1210877189000000 1210877189000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)<a name="note_1"></a>|@jml removed owner|
|-|-|

Merge approved.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set status to `closed`|
|-|-|

(In [23632]) Apply trial doc patch

Author: amacleod
Reviewer: jml
Fixes: [#2955](https://github.com/twisted/twisted/issues/2955)

Make the docstrings for the trial assertion method use consistent casing and fix
a some small factual errors in them.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
